autoibamr.sh

```diff
@@ -170,6 +170,7 @@ while [ -n "$1" ]; do
         # debug builds
         --enable-debugging)
             DEBUGGING=ON
+	    PREFIX=~/autoibamr-dbg
```

Comment:
```suggestion
	    PREFIX=~/autoibamr-debug
```
I'd prefer to avoid abbreviations - not everyone knows them.

Reply:
Actually, on second look - this isn't enough to work correctly. Since we parse this argument before we parse `--prefix`, it is possible that a user could do

```sh
./autoibamr.sh --prefix=$HOME/Applications/autoibamr --enable-debug
```
at which point their prefix will be overwritten. Could you check to see if this is the case? We should probably just adjust the default installation prefix in another way.

Reply:
this is intentional...

Reply:
I figured if someone provides their own prefix, that should override the default -- maybe they have a specific naming convention for the debug build? how do you think it should be done instead?

Reply:
Let me give a concrete example of what I think should not happen. Consider the following shell script:
```sh
#!/bin/bash

PREFIX=default
DEBUGGING=OFF

while [ -n "$1" ]; do
    param="$1"
    case $param in
        #####################################
        # debug builds
        --enable-debugging)
            DEBUGGING=ON
            PREFIX=~/autoibamr-debug
        ;;

        #####################################
        # Prefix path
        -p)
            shift
            PREFIX="${1}"
        ;;
        -p=*|--prefix=*)
            PREFIX="${param#*=}"
        ;;
    esac
    shift
done


echo "PREFIX = $PREFIX"
echo "DEBUGGING = $DEBUGGING"
```
and its execution:
```
[drwells@mitral experiments]$ ./prefix-test.sh --enable-debugging --prefix=test
PREFIX = test
DEBUGGING = ON
[drwells@mitral experiments]$ ./prefix-test.sh --prefix=test --enable-debugging
PREFIX = /afs/cas.unc.edu/users/d/r/drwells/autoibamr-debug
DEBUGGING = ON
```
i.e., the order of the flags impacts which output directory is ultimately selected. The order of the flags shouldn't impact the final parameter values.